CMakeLists.txt

```diff
@@ -60,8 +60,8 @@ endif()
 #   If the source code was changed, but there were no interface changes:
 #     Increment PATCH.
 set(LIBRARY_VERSION_MAJOR 16)
-set(LIBRARY_VERSION_MINOR 2)
-set(LIBRARY_VERSION_PATCH 1)
+set(LIBRARY_VERSION_MINOR 3)
+set(LIBRARY_VERSION_PATCH 0)
```

Comment:
The test pull request for the release artifacts: https://github.com/AOMediaCodec/libavif/pull/2792

I tested the avifdec and avifenc binaries on Linux, macOS, and Windows. I found a problem: the Windows artifacts ZIP file includes avifgainmaputil but the Linux and macOS files don't. I will write a pull request to fix this.

Reply:
Fixed in https://github.com/AOMediaCodec/libavif/pull/2794.